BambooTracker/res/lang/bamboo_tracker_fr.ts

```diff
@@ -958,52 +958,52 @@
     <message>
         <location filename="../../gui/mainwindow.ui" line="905"/>
         <source>Ctrl+I</source>
-        <translation></translation>
+        <translation>Ctrl+I</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="914"/>
         <source>&amp;Play</source>
-        <translation type="unfinished">Jouer</translation>
+        <translation>&amp;Jouer</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="923"/>
         <source>Play P&amp;attern</source>
-        <translation type="unfinished">Jouer le motif</translation>
+        <translation>Jouer le &amp;motif</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="926"/>
         <source>F6</source>
-        <translation></translation>
+        <translation>F6</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="931"/>
         <source>Play &amp;From Start</source>
-        <translation type="unfinished"></translation>
+        <translation>Jouer &amp;depuis le début</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="934"/>
         <source>F5</source>
-        <translation></translation>
+        <translation>F5</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="939"/>
         <source>Play From C&amp;ursor</source>
-        <translation type="unfinished"></translation>
+        <translation>Jouer depuis le c&amp;urseur</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="942"/>
         <source>F7</source>
-        <translation></translation>
+        <translation>F7</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="951"/>
         <source>&amp;Stop</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Stop</translation>
     </message>
     <message>
         <location filename="../../gui/mainwindow.ui" line="954"/>
         <source>F8</source>
-        <translation></translation>
+        <translation>f8</translation>
```

Comment:
Je remarque un erreur de casse ici.
On peut laisser vide quand la traduction est identique à l'original.

```suggestion
        <translation>F8</translation>
```

Reply:
J'aime pas laisser vide par habitude.
Mais fait comme tu veux.

Reply:
Oui je ne veux surtout pas t'embêter à tout remettre comme avant.
Il me semble que Qt ne met pas l'entrée dans le catalogue lorsque les 2 sont égales.

Cela dit je pense qu'en général il est bon de laisser vide. Je pense que les mêmes principes s'appliquent qu'en programmation. (1) la paresse est une vertu, pas la peine de produire soi-même un effort que la machine peut faire à la place (2) éviter toute redondance pouvant engendrer un risque d'erreur.